data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_eth_v1_beacon_light_client_updates.json

```diff
@@ -0,0 +1,96 @@
+{
+  "get" : {
+    "tags" : ["Beacon", "Experimental"],
+    "operationId" : "getLightClientUpdatesByRange",
+    "summary" : "Get `LightClientUpdate` instances in a requested sync committee period range",
+    "description" : "Requests the [`LightClientUpdate`](https://github.com/ethereum/consensus-specs/blob/v1.2.0-rc.3/specs/altair/light-client/sync-protocol.md#lightclientupdate) instances in the sync committee period range `[start_period, start_period + count)`, leading up to the current head sync committee period as selected by fork choice. Depending on the `Accept` header they can be returned either as JSON or SSZ-serialized bytes.",
+    "parameters" : [ {
+      "name" : "start_period",
+      "required" : true,
+      "in" : "query",
+      "schema" : {
+        "type" : "string",
+        "description" : "`uint64` value representing start_period",
+        "example" : "1",
+        "format" : "uint64"
+      }
+    }, {
+      "name" : "count",
+      "required" : true,
+      "in" : "query",
+      "schema" : {
+        "type" : "string",
+        "description" : "`uint64` value representing count",
+        "example" : "1",
+        "format" : "uint64"
+      }
+    } ],
+    "responses" : {
+      "200" : {
+        "description" : "Request successful",
+        "content" : {
+          "application/json" : {
+            "schema" : {
+              "title" : "GetLightClientUpdatesByRangeResponse",
+              "type" : "array",
+              "items" : {
+                "type" : "object",
+                "required" : [ "version", "data" ],
+                "properties" : {
+                  "version": {
+                    "type": "string",
+                    "enum": [
+                      "phase0",
+                      "altair",
+                      "bellatrix",
+                      "capella"
+                    ]
+                  },
+                  "data": {
+                    "$ref": "#/components/schemas/LightClientUpdate"
+                  }
+                }
+              }
+            }
+          },
```

Comment:
The structure here will be inconsistent with spec...
we've basically got 
```
[
{
 version: ...,
data: {...}
}
]
```
which i think the reference json shows too...

we actually need:
```
{
version: ...
data: [ {...}, {...}]
}
```

https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getLightClientUpdatesByRange
<img width="486" alt="Screenshot 2022-12-18 at 7 31 08 am" src="https://user-images.githubusercontent.com/2967240/208266720-77013a41-3157-49ea-a23a-17765d284f5d.png">